Docs: keyboard shortcuts. (#8442)
ckeditor/ckeditor5: docs/features/keyboard-support.md

# Keyboard support

> CKEditor 5 offers plenty of features that aid editing process and help streamline and speed up writing. Using keyboard shortcuts is one of the simplest yet most efficient and popular ways to improve interaction with the software.

Reviewer: ```suggestion
CKEditor 5 offers plenty of features to help you create your content in the most productive way. Using keyboard shortcuts is one of the simplest yet most efficient and popular ways to improve interaction with the software.
```

> Below is a list of the most important keystrokes supported by CKEditor 5 and its features:

Reviewer: ```suggestion
Below is a list of the most important keystrokes supported by CKEditor 5 and its features.
```

	width: 30%;
}
</style>

## Related productivity features

> There are more features that aid the efficiency and improve productivity:

Reviewer: ```suggestion
In addition to using keyboard shortcuts, you may want to check the following productivity features:
```
